File: areatriangulo.cpp
#include <iostream>
int main () {
//Definir Variables y otros
printf("Ejercicio 01: Area de un Triangulo\n");
int b=0, h=0, area=0;
//Datos de Entrada
printf("Ingrese Base:");
scanf("%i", &b);
printf("Ingrese Altura:");
scanf("%i", &h);
//Proceso
area=(b*h)/2;
//Datos de Salida
printf("Area de Triangulo es:%i %s", area, "\n");
return 0;
}

File: distint-language/areatriangulo.kt
fun main() {
//Definir Variables y otros
println("Ajercicio 01: Area de un Triangulo")
var b: Int?
var h: Int?
var area: Int?
//Datos de Entrada
println("Ingrese Base:")
b=readLine()?.toIntOrNull()!!
println("Ingrese Altura:")
h=readLine()?.toIntOrNull()!!
//Proceso
area=(b*h)/2
//Datos de salida
println("El Area del Triangulo es:"+area)
}